tests/appsec/rasp/test_stack_traces.py

```diff
@@ -62,6 +62,7 @@ class Test_StackTrace:
     def setup_lfi_stack_trace(self):
         self.r = weblog.get("/rasp/lfi", params={"file": "../etc/passwd"})
 
+    @missing_feature(library="nodejs")
```

Comment:
I think we should leave this modification for another PR (the one for enabling stack traces test for nodejs) and keep this PR just for metrics.

Reply:
Yes, you're right.